app/middleware.py

```diff
@@ -1,4 +1,4 @@
-class ReverseProxied(object):
+class ReverseProxies(object):
```

Comment:
why plural?

I don't like too much black magic done in this ReverseProx(ied|ies|y) class, until we have a stable pattern for service stack/ routing. I'd leave this to every service to decide...

Reply:
I don't know why plural but it was not consistent, the reference to the class was done using ReverseProxies but the class was defined ReversProxied. However I could change it overall to ReverseProxy ? I also prefer singular :-)